### PR TITLE
feature branch

### PR DESCRIPTION
**Pull request by Pieta9221**
@anicheemmanuel90@gmail.com Please review my code

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -49,4 +49,13 @@ table {
...
\ No newline at end of file
+/* Write your css below here */
+
+h1{
+ color: blue;
+}
+
+
+.char-man{
+	color: blue;
```
Comment (Melumm) [inline css/style.css:60]: please remove the white spaces

File: html/about.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>About</title>
+    </head>
+    <body>
+        <nav>
+            <a href="./index.html">Home</a>
+            <a href="./about.html">About</a>
+            <a href="./contact.html">Contact</a>
+        </nav>
+        <h1>This is Home page</h1>
```
Comment (Melumm) [inline html/about.html:15]: change title to about

File: html/contact.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Document</title>
```
Comment (Melumm) [inline html/contact.html:7]: Add a description pls

**Review by Melumm — COMMENTED** [review]
Do this before friday 12pm

**Comment by Melumm** [comment]
⭐⭐⭐
Congratulation! You are superb!!!

**Comment by Melumm** [comment]
Please go ahead and merge...

**Comment by Pieta9221** [comment]
I  appreciate sir

**Pull request merged**